### PR TITLE
GUVNOR-2782: Solve marshalling issues of ImpossibleMatchIssue

### PR DESCRIPTION
**Pull request by jomarko**
After reading Errai Marshalling wiki it seems to me that I solved the problem, however I was not able to test appropriately the changes because of GWT stuff. Trying to test better. Please do not merge yet.

### CODE REVIEW

**Comment by Rikkola** [comment]
@jomarko Hi, is this bug still an issue? I could take a quick look, looks like we could get it fixed with some effort.

**Comment by jomarko** [comment]
@Rikkola I added `ReceiverErraiTest` however this fails because `MarshallingWrapper` returns empty string, could you please help me with this? I hope it is just some setup issue of the test, did I miss something in the added gwt.xml?

**Comment by manstis** [comment]
@jomarko @Rikkola Please advise if/when this can be merged.

File: drools-wb-services/drools-wb-verifier/drools-wb-verifier-api/src/main/java/org/drools/workbench/services/verifier/api/client/reporting/ImpossibleMatchIssue.java [hunk 1]
```diff
@@ -38,10 +36,10 @@ public ImpossibleMatchIssue( @MapsTo("severity") final Severity severity,
                                  @MapsTo("fieldName") final String fieldName,
                                  @MapsTo("conflictedItem") final String conflictedItem,
                                  @MapsTo("conflictingItem") final String conflictingItem,
-                                 @MapsTo("rowNumbers") final Integer... rowNumbers ) {
+                                 @MapsTo("rowNumbers") final Set<Integer> rowNumbers ) {
```
Comment (Rikkola) [inline drools-wb-services/drools-wb-verifier/drools-wb-verifier-api/src/main/java/org/drools/workbench/services/verifier/api/client/reporting/ImpossibleMatchIssue.java]: Ok, so is this the actual fix?

**Comment by Rikkola** [comment]
@jomarko Tried my best to test this, but the only way is to use GWT Test Case and we should avoid that. I made a ticket for the proper test. This PR does fix the issue so we just need to clean it up and get it in. For this to happen we need to ignore the fact that the test coverage will be incomplete in drools-wb. 

https://issues.jboss.org/browse/GUVNOR-3071

**Comment by jomarko** [comment]
@Rikkola this PR updated according our irc discussion.

**Comment by jomarko** [comment]
@manstis it seems to me as needed also in 7.0.x, what is your opinion?

**Comment by manstis** [comment]
@jomarko Yes, I agree... let's close as many holes as possible.

**Pull request merged**